clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-infra_machine_set.yaml

```diff
@@ -16,8 +16,8 @@ spec:
       creationTimestamp: null
       labels:
         machine.openshift.io/cluster-api-cluster: build01-9hdwj
-        machine.openshift.io/cluster-api-machine-role: worker
-        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machine-role: infra
```

Comment:
Is it needed only for healthCheck?
Or something we are supposed to do but did not in the past?

Reply:
OSD has both infra nad worker so i guess its better to set up both.